modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-32-0_{context}"]
+= Red Hat {ServerlessProductName} 1.32
+
+{ServerlessProductName} 1.32 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-32-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.11.
+* {ServerlessProductName} now uses Knative Eventing 1.11.
+* {ServerlessProductName} now uses Kourier 1.11.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.11.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.11.
+* The `kn func` CLI plug-in now uses `func` 1.11.
```

Comment:
Where did you get the version for `func`? It's not in the YAML, so I would ask the team to specifically check this version.

Reply:
Yes, correct it is not in the YAML. Should I ask the team directly in the channel regarding the version?

Reply:
The `func` is 1.13 in this release.